practice-1/print_word_avoids_letter.py

```diff
@@ -0,0 +1,24 @@
+"""Write a function named avoids that takes a word and a string of forbidden letters,
+# and that returns True if the word doesn’t use any of the forbidden letters."""
+def avoids_letter():
+    words = 'hello@'
+    forbidden_letter = '!@#$%*&'
+    for letter in forbidden_letter:
+        if letter in words:
+            return False
+    return True
+results = avoids_letter()
+print(results)  
+
+# """Modify your program to prompt the user to enter a string of forbidden letters and
```

Comment:
Pay attention to the single-line and multiple-line comment syntax.
Please check for all.